### PR TITLE
EZTV provider

### PR DESCRIPTION
**Pull request by nicolasmartinelli**
This is a EZTV provider using the PopcornTime API http://eztvapi.re/. I started working on it since there is no EZRSS feed working as of today (ezrss.ch, ezrss.it and ezrss-proxy.net are down). 

The provider only supports a search mode by episode. This is by design, since the PopcornTime API does not provide a RSS feed, and that EZTV does not provide season packs. So far, it works fine, but some adjustments will probably be necessary.

Any feedback welcome :)

### CODE REVIEW

**Comment by nicolasmartinelli** [comment]
I've been using it for a couple of days without issues. Any feedback? Any interest in the provider?

**Comment by abeloin** [comment]
@nicolasmartinelli Please rebase

**Comment by abeloin** [comment]
@fernandog Merging for v4.0.14?

**Comment by fernandog** [comment]
@abeloin where is the settting backlog search not  available?

**Comment by fernandog** [comment]
@nicolasmartinelli please rebase

**Comment by nicolasmartinelli** [comment]
Rebase is done.
@fernandog: no warning about backlog search.

**Comment by fernandog** [comment]
@abeloin how that "**" appears next to provider in provider list so user can know backlog is not possible?

**Comment by nicolasmartinelli** [comment]
@fernandog : just to be sure, but what exactly is backlog?

**Comment by nicolasmartinelli** [comment]
Just to make it clear: this EZTV provider is different from EZRSS. EZRSS is just a RSS feed, so links disappear with time. It is not the case of the EZTV provider: it allows browsing in the entire catalogue of EZTV, so the links never disappear.
I could easily download files from a couple of years ago with it.

File: sickbeard/providers/eztv.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+import generic
+from sickbeard import logger, tvcache, db
+from sickbeard.common import Quality
+
+class EZTVProvider(generic.TorrentProvider):
+
+    def __init__(self):
+        generic.TorrentProvider.__init__(self, "EZTV")
+
+        self.supportsBacklog = True
```
Comment (abeloin) [inline sickbeard/providers/eztv.py]: This need to be set to false as the provider doesn't support backlog.

Comment (nicolasmartinelli) [inline sickbeard/providers/eztv.py]: OK, done

**Comment by abeloin** [comment]
@nicolasmartinelli Can you squash your commit into one?

**Comment by nicolasmartinelli** [comment]
@abeloin : the branch is rebased and the commit is squashed.

Thanks!

**Comment by fernandog** [comment]
@nicolasmartinelli can you check this one?

https://github. com/SiCKRAGETV/sickrage-issues/issues/1106

**Comment by abeloin** [comment]
@fernandog Fixed in #1712

**Comment by nicolasmartinelli** [comment]
Thanks for the fix :)

**Pull request merged**